nion/swift/Panel.py

```diff
@@ -376,6 +395,108 @@ def context_menu_event(self, x: int, y: int, gx: int, gy: int) -> bool:
             return self.on_context_menu_clicked(x, y, gx, gy)
         return False
 
+
+    def _draw_controls_windows(self, drawing_context: DrawingContext.DrawingContext) -> None:
+        self.__close_control_rect = self._draw_close_control_windows(drawing_context)
+        title_rect = self._draw_title_text(drawing_context)
+        if title_rect and title_rect.width > 0:
+            self.__edit_control_rect = self._draw_edit_button(drawing_context, title_rect)
+        else:
+            self.__edit_control_rect = Geometry.IntRect.from_tlhw(0, 0, 0, 0)
+
+    def _draw_close_control_mac(self, drawing_context: DrawingContext.DrawingContext) -> None:
+        self._draw_close_control_windows(drawing_context)
+    def _draw_close_control_linux(self, drawing_context: DrawingContext.DrawingContext) -> None:
```

Comment:
Should be a single blank line here. Please consider using `flake8 --ignore=E501 nion/swift/Panel.py nion/swift/DisplayPanel.py` and addressing any issues specific to this PR. You may need to install `conda install flake8` in your environment.